lib/stages/basebandReadout.cpp

```diff
@@ -627,7 +628,7 @@ void basebandReadout::write_dump(basebandDumpData data, basebandDumpStatus& dump
         if (ii_samp >= data.data_length_fpga)
             break;
         // Add intentional throttling.
-        float stime = _write_throttle * to_write * FPGA_PERIOD_NS;
+        float stime = _write_throttle * to_write * delta_t * 1e9;
```

Comment:
I think this would then also be more readable if `delta_t` was named `fpga_s` -- since originally "delta_t" variable was used just to get the value for the "delta_time" attribute for the baseband dump file metadata.